python/django/python-hello-world/README.md

```diff
@@ -0,0 +1,72 @@
+# Hello World with Cloud Code
+
+![Architecture Diagram](./img/diagram.png)
```

Comment:
This diagram is great for hello-world use-case but you also have database. I suspect that you mixed two templates into one:
 - hello-world sample should only print "hello world" and have no dependencies.
 - guestbook sample can depend on postgres or other storage and allow storing and retrieving guestbook records